### PR TITLE
Serval 13 (serw13)

### PR DESCRIPTION
**Pull request by jacobgkau**
This adds the Serval WS 13 (serw13) to tech docs.

### CODE REVIEW

File: src/models/serw13/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    - Internal speakers & microphone
+    - Combined headphone & microphone 3.5mm jack
+    - Combined microphone & S/PDIF 3.5mm jack
+    - HDMI, Mini DisplayPort, USB-C DisplayPort audio
+- Storage
+    - 2x PCIe NVMe Gen 4
+    - MicroSD card reader
+- USB
+    - 1x USB Type-C with Thunderbolt 4
+        - Supports USB-PD (charging) and DisplayPort over USB-C.
```
Comment (ahoneybun) [inline src/models/serw13/README.md:58]: How did we test this? USB-PD only goes up to 240W with the new current spec.

File: src/models/serw13/README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    - GigaDevice GD25B256EYIGR
+        - WSON-8 form factor
+    - Programmed with [System76 Open Firmware](https://github.com/system76/firmware-open)
+- EC
+    - ITE IT5570E
+    - Programmed with [System76 EC](https://github.com/system76/ec)
+- Graphics
+    - GPU options:
+        - NVIDIA GeForce RTX 4050
+        - NVIDIA GeForce RTX 4060
```
Comment (ahoneybun) [inline src/models/serw13/README.md:28]: I don't see a 4050 option on this page and there isn't a 4K option listed here either.

Comment (leviport) [reply src/models/serw13/README.md:28]: The serw13 will only come with 4060 and 4070 options. The addw3 will have all three.

File: src/models/serw13/README.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        - NVIDIA GeForce RTX 4070
+    - eDP display: 15.6" 1920x1080@144Hz LCD
```
[the remark this answers is not in the document]
Comment (leviport) [reply src/models/serw13/README.md:30]: Correct, 15" is FHD at 165Hz, and the 17" is 4k at 144Hz

File: src/models/serw13/README.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+
+- [External Overview](./external-overview.md)
+- [Internal Overview](./internal-overview.md)
+- [Parts & Repairs](./repairs.md)
+
+![Serval WS](./img/serw13.webp)
+
+The System76 Serval WS is a laptop with the following specifications:
+
+- CPU
```
[the remark this answers is not in the document]
Comment (leviport) [reply src/models/serw13/README.md:11]: No, serw13 will only have the i9-13900HX

File: src/models/serw13/README.md (hunk 1 above)
Comment (jacobgkau) [reply src/models/serw13/README.md:58]: I should probably add a note that it's only enabled when the machine's powered off.

File: src/models/serw13/README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (jacobgkau) [reply src/models/serw13/README.md:11]: Addressed: 01fab975c650a909587272464782c76323bbabde

File: src/models/serw13/README.md (hunk 1 above)
Comment (jacobgkau) [reply src/models/serw13/README.md]: Addressed: 732cb7d82b32e56ea7b6c1384eed4bbde6426417

File: src/models/serw13/README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (jacobgkau) [reply src/models/serw13/README.md]: Addressed: d605e46e9eec620286a513aec1d30f991ad62d49

File: src/models/serw13/repairs.md [hunk 5]
```diff
@@ -0,0 +1,243 @@
...
+2. Pull the keyboard adhesive out of the machine to detatch it from the keyboard.
+    - The adhesive strip access point is highlighted green below.
+    - If the end of the keyboard adhesive strip is tucked behind the fan, it can be pulled out using tweezers or a small screwdriver, or the [thermal system can be removed](#replacing-the-cooling-system) to expose it.
+        - Replacing the thermal paste is recommended if the thermal system is removed.
+    - If the adhesive strip breaks, remove as much of it as possible. The keyboard can be reinstalled without the adhesive strip.
+
+![Keyboard adhesive & push point](./img/keyboard-adhesive.webp)
+
+3. Open the lid slightly and place the machine on its side.
+4. Push the screwdriver into the keyboard push point (highlighted cyan above) until the keyboard pops out.
```
Comment (leviport) [inline src/models/serw13/repairs.md:74]: > (highlighted cyan above)

s/cyan/red

File: src/models/serw13/repairs.md [hunk 6]
```diff
@@ -0,0 +1,243 @@
...
+1. Follow the steps above to [remove the bottom cover](#removing-the-bottom-cover) and [outermost SSD](#replacing-an-m2nvme-ssd).
+2. Remove the [innermost SSD](#replacing-an-m2nvme-ssd) and [remove the wireless card](#replacing-the-wireless-card), then pull the battery away from the adhesive holding it to the case.
+    - If you are not replacing the CMOS battery, then this step is optional.
+
+![CMOS battery](./img/cmos-battery.webp)
+
+3. Unplug the white connector that connects the CMOS battery to the motherboard.
+4. To clear the CMOS, open the lid of the machine and hold down the power button for at least 15 seconds to discharge any residual energy in the system.
+5. Re-connect the CMOS battery.
+6. Replace the bottom panel and main battery.
```
Comment (leviport) [inline src/models/serw13/repairs.md:209]: Replacing the main battery is mentioned, but unplugging or removing it isn't mentioned prior. It doesn't look like fully removing the main battery is necessary, but unplugging it before trying to clear CMOS should probably be mentioned.

Comment (jacobgkau) [reply src/models/serw13/repairs.md]: cb88769edd999a58d754f7b9586cf00b8da76e9e should make a little more sense.

**Review by leviport — APPROVED** [review]
Great work!

**Pull request merged**